txconnpool/pool.py

```diff
@@ -183,15 +221,14 @@ def _connected(client):
 
         self._factories.append(factory)
 
-        self._reactor.connectTCP(self._serverAddress.host,
-                                 self._serverAddress.port,
+        self._reactor.connectTCP(self._nextServerAddress.host,
```

Comment:
better to store `self._nextServerAddress` in a local variable and get its host and port, than accessing `self._nextServerAddress` twice to get `host` and `port`